### PR TITLE
35 landing page services functionality only

### PR DESCRIPTION
**Pull request by Andy-Reynolds**

### CODE REVIEW

**Review by DanForder — CHANGES_REQUESTED** [review]
fix PR build issues

File: src/components/ServicesCard/ServicesCard.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import "./ServicesCard.scss";
+import vectorBlack from "../../assets/svgs/Vector-black.svg";
+import vectorWhite from "../../assets/svgs/Vector-white.svg";
+import { Link } from "react-router-dom";
+
+const ServicesCard = ({ title, description }) => {
+  return (
+    <div className="services-card">
+      <h1 className="services-card__title">{title}</h1>
```
Comment (DanForder) [inline src/components/ServicesCard/ServicesCard.jsx:9]: There should only ever be one H1 on each page.

Think about the hierarchy of your headings on this page. Each heading should make sense semantically underneath it's parent. 

For example:

h1 : Services
h2: Services I offer
h3: Online Courses, h3: Bespoke Courses, h3 Group Courses

File: src/components/ServicesCard/ServicesCard.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <h2>{description}</h2>
```
Comment (DanForder) [inline src/components/ServicesCard/ServicesCard.jsx:10]: The description is probably best as `<p>` tag

File: src/components/ServicesCard/ServicesCard.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      <Link to="/services">
+        <div className="services-card__button">
+          <img
+            className="services-card__vector services-card__vector--black"
+            src={vectorBlack}
+            alt=""
+          />
+          {/* Put a second chevron here of a different colour with display=none. We can use this to change the colour when we add hover later */}
```
Comment (DanForder) [inline src/components/ServicesCard/ServicesCard.jsx:18]: remove comment

File: src/components/ServicesCard/ServicesCard.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import vectorWhite from "../../assets/svgs/Vector-white.svg";
+import { Link } from "react-router-dom";
+
+const ServicesCard = ({ title, description }) => {
+  return (
+    <div className="services-card">
+      <h1 className="services-card__title">{title}</h1>
+      <h2>{description}</h2>
+      <Link to="/services">
+        <div className="services-card__button">
```
Comment (DanForder) [inline src/components/ServicesCard/ServicesCard.jsx:12]: can you remove this div and put the className on the `Link` component? could remove a bit of unnecessary html

File: src/components/ServicesCard/ServicesCard.scss [hunk 5]
```diff
@@ -0,0 +1,30 @@
+@use "../../styles/variables" as *;
+.services-card {
+  background-color: #e4f5f3;
+  max-width: 23.125rem;
+  //when all content is in, this can be changed to max height
```
Comment (DanForder) [inline src/components/ServicesCard/ServicesCard.scss:5]: remove comment/ max-height is probably better too

File: src/components/ServicesCard/ServicesCard.scss [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    height: 2.8rem;
+    width: 2.8rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 1rem;
+  }
+  &__vector {
+    &--white {
+      display: none;
```
Comment (DanForder) [inline src/components/ServicesCard/ServicesCard.scss:27]: if this is always hidden do we need it at all in the html?

File: src/components/ServicesSection/ServicesSection.jsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+          ipsum saepe ipsa optio dolorem veniam tenetur nemo labore ut"
+        />
+        <ServicesCard
+          title="Bespoke Packages"
+          description=" Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quia quod
+          ipsum saepe ipsa optio dolorem veniam tenetur nemo labore ut"
+        />
+      </div>
+      <Button
+        className="services-section__button"
```
Comment (DanForder) [inline src/components/ServicesSection/ServicesSection.jsx:31]: does Button have a prop called className that you're using?

File: src/components/ServicesSection/ServicesSection.scss [hunk 8]
```diff
@@ -0,0 +1,36 @@
+@use "../../styles/variables" as *;
+.services-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 2rem;
+  .button--secondary {
```
Comment (DanForder) [inline src/components/ServicesSection/ServicesSection.scss:7]: we shouldn't be modifying the button component like this from outside the button styles. You could create a clasname prop on button if required

File: src/components/ServicesSection/ServicesSection.jsx [hunk 9]
```diff
@@ -0,0 +1,40 @@
+import Button from "../Button/Button";
+import ServicesCard from "../ServicesCard/ServicesCard";
+import "./ServicesSection.scss";
+
+const ServicesSection = ({ link }) => {
```
Comment (DanForder) [inline src/components/ServicesSection/ServicesSection.jsx:5]: does link need to be a prop? don't think this is going to change so might be okay to be hard coded on line 34

File: src/components/ServicesCard/ServicesCard.jsx (hunk 3 above)
Comment (Andy-Reynolds) [reply src/components/ServicesCard/ServicesCard.jsx:18]: Can we leave this comment for now as a reference for the next person to pick up the ticket?

**Comment by Andy-Reynolds** [comment]
Duplicate Branch - closed request

**Pull request closed without merging**